### PR TITLE
add page with mirrors sites and link on home page

### PR DESCRIPTION
**Pull request by mlennert**
First attempt at adding a list of mirror sites.
Current list certainly not up to date, so needs to be corrected before release.

I added the link only on the home page. Not sure if it should go somewhere else.

The actual page is in content/about/mirrors as we have no content directly in content/ and so I didn't want to put it there. About seemed the best for now.

### CODE REVIEW

**Comment by mlennert** [comment]
This a fix for #181

File: content/about/mirrors.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "GRASS Website mirrors"
+date: 2020-07-28T16:13:30+02:00
+layout: "general"
+---
+
+### Official master site via HTTP/FTP
+
+
+* [Official master site: grass.osgeo.org](http://grass.osgeo.org/)
```
Comment (neteler) [inline content/about/mirrors.md:10]:
```suggestion
* [Official master site: grass.osgeo.org](https://grass.osgeo.org/)
```

File: content/about/mirrors.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* We recommend cascaded mirroring: 
+
+  * Tier-1 mirrors: These mirrors read daily from the master site and might also act as rsync servers. 
+  * Tier-2 mirrors: These mirrors read from the tier-1 mirrors (these sites may be 1-2 day(s) behind the master site)
```
Comment (neteler) [inline content/about/mirrors.md:15]: This is no longer needed (may be removed)

Comment (mlennert) [reply content/about/mirrors.md]: Ok, I've removed it.

**Comment by mlennert** [comment]
Can I merge this ?

File: content/about/mirrors.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: "GRASS Website mirrors"
```
Comment (neteler) [inline content/about/mirrors.md:2]:
```suggestion
title: "GRASS GIS Website mirrors"
```

**Comment by mlennert** [comment]
@neteler  Just noticed that https://namesdir.com/mirrors/grass/ seems no longer active. Do you anything about this mirror ?

**Comment by neteler** [comment]
> @neteler Just noticed that https://namesdir.com/mirrors/grass/ seems no longer active. Do you anything about this mirror ?

No nothing, let's remove the entry for now.

**Comment by mlennert** [comment]
> > @neteler Just noticed that https://namesdir.com/mirrors/grass/ seems no longer active. Do you anything about this mirror ?
> 
> No nothing, let's remove the entry for now.

Done in 1d250c090f32d815e4c1fea4f63259634f7ca9ce 
(plus added hint that master site is in the USA)

**Pull request merged**